src/ConnectedMode.UnitTests/Migration/FileProviders/XmlFileProviderTests.cs

```diff
@@ -177,5 +234,42 @@ private static Mock<IVsSolution> CreateIVsSolution(string pathToReturn)
 
             return solution;
         }
+
+        private static Mock<IRoslynProjectProvider> CreateRoslynProjectProvider(params string[] projectFilePaths)
+        {
+            var projectProvider = new Mock<IRoslynProjectProvider>();
+
+            var projects = CreateProjects(projectFilePaths);
+            projectProvider.Setup(x => x.Get()).Returns(projects);
+
+            return projectProvider;
+        }
+
+        private static IReadOnlyList<Project> CreateProjects(params string[] projectFilePaths)
```

Comment:
These helper methods aren't the same as the ones I wrote for the `RoslynProjectProvider` itself i.e. we can share them.

These tests need to mock the project file path only.